Java/core/src/main/java/com/amazon/randomcutforest/store/PointStore.java

```diff
@@ -710,6 +689,11 @@ public T capacity(int capacity) {
             return (T) this;
         }
 
+        public T initialSize(int initialPointStoreSize) {
+            this.initialPointStoreSize = initialPointStoreSize;
+            return (T) this;
+        }
+
         public T currentStoreCapacity(int currentStoreCapacity) {
```

Comment:
I wonder if we should change `capacity` to `maxCapacity` for the point store, and use `capacity` to mean "current capacity", i.e., number of points that can be stored without resizing. What do you think?

Reply:
maxCapacity sounds like "active consideration" -- capacity is already signaling "max". But we can change if desired